norduniclient/models.py

```diff
@@ -189,6 +191,29 @@ def delete(self):
     def reload(self, node=None):
         return core.get_node_model(self.manager, self.handle_id, node=node)
 
+    def add_property(self, property, value):
+        if isinstance(value, six.string_types):
+            value = "'{}'".format(value)
+
+        q = """
+            MATCH (n:Node {{handle_id: {{handle_id}}}})
+            SET n.{property} = {value}
+            RETURN n
+            """.format(property=property, value=value)
```

Comment:
Not that it matters a lot (and our code is probably full of these things all over the place), but this is not injection safe.

Reply:
I don't think setting properties ever was injection safe (and I think you are correct that the problem can be found elsewhere to).